integration/utils.js

```diff
@@ -0,0 +1,35 @@
+const axios = require("axios");
+
+const axiosOptions = {
+  headers: {
+    "User-Agent": "Node",
```

Comment:
praise: thank you for the custom header! 
nit: maybe a better custom name would be "X-Dotcom-Integration"

Reply:
`X-...` is the convention for the name of a custom header, not the value. We're distinguishing between `Node`, `Artillery`, and `Playwright`. We can change `Node` to `Monitor` or something but the idea is to name the `User-Agent` with the agent which is just a node application.